<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SIOverviewDetails</class>
 <widget class="QWidget" name="SIOverviewDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBoxDetails">
     <property name="title">
      <string>Details :</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
        </property>
        <property name="rowWrapPolicy">
         <enum>QFormLayout::RowWrapPolicy::DontWrapRows</enum>
        </property>
        <property name="labelAlignment">
         <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
        </property>
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="editServiceName">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Category:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Service Category:</string>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0" colspan="2">
            <widget class="QWidget" name="widget" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QRadioButton" name="radioPublic">
                <property name="palette">
                 <palette>
                  <active>
                   <colorrole role="Accent">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>240</red>
                      <green>240</green>
                      <blue>240</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </active>
                  <inactive>
                   <colorrole role="Accent">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>240</red>
                      <green>240</green>
                      <blue>240</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </inactive>
                  <disabled>
                   <colorrole role="Accent">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>240</red>
                      <green>240</green>
                      <blue>240</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </disabled>
                 </palette>
                </property>
                <property name="text">
                 <string>Public</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioPrivate">
                <property name="palette">
                 <palette>
                  <active>
                   <colorrole role="Accent">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>240</red>
                      <green>240</green>
                      <blue>240</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </active>
                  <inactive>
                   <colorrole role="Accent">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>240</red>
                      <green>240</green>
                      <blue>240</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </inactive>
                  <disabled>
                   <colorrole role="Accent">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>240</red>
                      <green>240</green>
                      <blue>240</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </disabled>
                 </palette>
                </property>
                <property name="text">
                 <string>Private</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioInternet">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="palette">
                 <palette>
                  <active>
                   <colorrole role="Accent">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>240</red>
                      <green>240</green>
                      <blue>240</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </active>
                  <inactive>
                   <colorrole role="Accent">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>240</red>
                      <green>240</green>
                      <blue>240</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </inactive>
                  <disabled>
                   <colorrole role="Accent">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>240</red>
                      <green>240</green>
                      <blue>240</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </disabled>
                 </palette>
                </property>
                <property name="text">
                 <string>Internet</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Description:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPlainTextEdit" name="textDescribe">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="plainText">
           <string notr="true"/>
          </property>
          <property name="overwriteMode">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string notr="true">Describe data here</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="checkDeprecated">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Deprecated:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="editDeprecated">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Version:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Interface Version:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="4">
            <widget class="QLineEdit" name="editVersionPatch">
             <property name="layoutDirection">
              <enum>Qt::LayoutDirection::RightToLeft</enum>
             </property>
             <property name="inputMask">
              <string>99999</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Major</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>.</string>
             </property>
             <property name="margin">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="0" column="4">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Patch</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Minor</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="editVersionMajor">
             <property name="layoutDirection">
              <enum>Qt::LayoutDirection::RightToLeft</enum>
             </property>
             <property name="inputMask">
              <string>99999</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLineEdit" name="editVersionMinor">
             <property name="layoutDirection">
              <enum>Qt::LayoutDirection::RightToLeft</enum>
             </property>
             <property name="inputMask">
              <string>99999</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>.</string>
             </property>
             <property name="margin">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioPublic</tabstop>
  <tabstop>radioPrivate</tabstop>
  <tabstop>radioInternet</tabstop>
  <tabstop>editVersionMajor</tabstop>
  <tabstop>editVersionMinor</tabstop>
  <tabstop>editVersionPatch</tabstop>
  <tabstop>textDescribe</tabstop>
  <tabstop>checkDeprecated</tabstop>
  <tabstop>editDeprecated</tabstop>
  <tabstop>editServiceName</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
